src/components/icons/svgIcons.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const ChevronRightIcon = (props) => (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" height={size}>
+        <path d="M10.5 5.5l5 5-5 5" fill={props.color}/>
+    </svg>
+);
+
+const ChevronLeftIcon = () => (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path d="M13.5 5.5l-5 5 5 5" fill={color}/>
+    </svg>
+);
```

Comment:
since the `svg` element is the same for all icons AND you may want to customize some attributes (like size, color, etc) when you use `FeatherIcon`, you _could_  just have this component output all elements nested in `<svg>`. also would improve maintainability so you don't have to update the `FeatherIcon` each time you want to add a new icon to use. ideally, you'd just add it to this file and specify the name in the prop of `FeatherIcon` wherever you use it

example of this pattern: https://github.com/roadlittledawn/portfolio-site/blob/develop/src/components/Icons/feather.js